multidc/README.md

```diff
@@ -10,6 +10,8 @@ Running multi dc tests has three stages, automating to varying degrees:
 
 Creation of the hosts is not automated (yet).
 
+Make sure you have the AWS CLI: `pip install awscli`.
```

Comment:
I'm taking notes of all the things, and will update when I got everything working

Reply:
Ok, awesome 👍